js_modules/dagit/packages/core/src/workers/runLogs/runLogs.worker.ts

```diff
@@ -0,0 +1,42 @@
+import type {Message} from './runLogs';
+
+type SHUTDOWN = {
+  type: 'SHUTDOWN';
+  staticPathRoot?: undefined;
+};
```

Comment:
Define `INITIALIZE` and `SHUTDOWN` types in the same location in a union so that you can have `data` be a single union type? Just to tidy things up a bit.